types/react/index.d.ts

```diff
@@ -284,7 +285,7 @@ declare namespace React {
         // tslint:enable:unified-signatures
 
         forceUpdate(callBack?: () => any): void;
-        render(): JSX.Element | null | false;
```

Comment:
Also add string, number, now that those are supported?

Reply:
Done. Here's the more complete docs: https://reactjs.org/docs/react-component.html#render